<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/parser/recipe_parser.py", line 117, in _parse_yaml
    output = cast(JsonType, yaml.safe_load(s))
                            ^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/composer.py", line 35, in get_single_node
    if not self.check_event(StreamEndEvent):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/parser.py", line 142, in parse_implicit_document_start
    if not self.check_token(DirectiveToken, DocumentStartToken,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/scanner.py", line 258, in fetch_more_tokens
    raise ScannerError("while scanning for the next token", None,
yaml.scanner.ScannerError: while scanning for the next token
found character '%' that cannot start any token
  in "<unicode string>", line 1, column 2:
    {% set pkg_src = ('src/'+goname). ...
     ^

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 728, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 834, in lint_file
    self.lint_recipe(
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 894, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/check_syntax.py", line 35, in check_recipe
    ro_parser: Final[RecipeParser] = recipe.get_read_only_parser()
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 675, in get_read_only_parser
    return RecipeParser("\n".join(self.meta_yaml))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/parser/recipe_parser.py", line 328, in __init__
    new_node = RecipeParser._parse_line_node(clean_line)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/parser/recipe_parser.py", line 147, in _parse_line_node
    output = RecipeParser._parse_yaml(s)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/parser/recipe_parser.py", line 126, in _parse_yaml
    cast(JsonType, yaml.safe_load(s)), lambda d: substitute_markers(d, sub_list)
                   ^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/composer.py", line 35, in get_single_node
    if not self.check_event(StreamEndEvent):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/parser.py", line 142, in parse_implicit_document_start
    if not self.check_token(DirectiveToken, DocumentStartToken,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/yaml/scanner.py", line 258, in fetch_more_tokens
    raise ScannerError("while scanning for the next token", None,
yaml.scanner.ScannerError: while scanning for the next token
found character '%' that cannot start any token
  in "<unicode string>", line 1, column 2:
    {% set pkg_src = ('src/'+goname). ...
     ^

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_ab9x1rng3w/clone` failed. (See above for error)
</details>

--- a/recipe/conda_build_config.yaml
+++ b/recipe/conda_build_config.yaml
@@ -1,2 +1,0 @@
-go_compiler:
-  - go-nocgo

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -2,28 +2,33 @@
 {% set version = "0.55.0" %}
 
 {% set name = goname.split('/')[-1] %}
-{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}  # src/github.com/junegunn/fzf
 
 package:
   name: {{ name|lower }}
   version: {{ version|replace("-", "_") }}
 
 source:
-  - folder: {{ pkg_src }}
-    url: https://{{ goname }}/archive/v{{ version }}.tar.gz
-    sha256: 805383f71bca7f8fb271ecd716852aea88fd898d5027d58add9e43df6ea766da
+  folder: {{ pkg_src }}
+  url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+  sha256: 805383f71bca7f8fb271ecd716852aea88fd898d5027d58add9e43df6ea766da
 
 build:
   number: 0
+  skip: true  # [s390x]
   script:
     - pushd {{ pkg_src }}
-    - go build -ldflags "-X main.revision=conda-forge" -v -o $PREFIX/bin/fzf  # [not win]
-    - go build -ldflags "-X main.revision=conda-forge" -v -o %LIBRARY_BIN%\fzf.exe  # [win]
-    - bash -c "mkdir -p ${PREFIX}/share/fzf/shell; cp -v shell/* ${PREFIX}/share/fzf/shell/"
-    - bash -c "mkdir -p ${PREFIX}/share/fzf/plugin; cp -v plugin/* ${PREFIX}/share/fzf/plugin/"
-    - go-licenses save . --save_path ../../../../library_licenses
+    - go build -ldflags "-X main.revision=anaconda" -v -o %LIBRARY_BIN%\fzf.exe                      # [win]
+    - mkdir %PREFIX%\share\fzf                                                                       # [win]
+    - xcopy /s /y shell %PREFIX%\share\fzf\shell\                                                    # [win]
+    - xcopy /s /y plugin %PREFIX%\share\fzf\plugin\                                                  # [win]
+    - go-licenses.exe save . --save_path %SRC_DIR%\\library_licenses                                 # [win]
+    - go build -ldflags "-X main.revision=anaconda" -v -o $PREFIX/bin/fzf                            # [not win]
+    - mkdir -p ${PREFIX}/share/fzf/shell; cp -v shell/* ${PREFIX}/share/fzf/shell/                   # [not win]
+    - mkdir -p ${PREFIX}/share/fzf/plugin; cp -v plugin/* ${PREFIX}/share/fzf/plugin/                # [not win]
+    - go-licenses save . --save_path ${SRC_DIR}/library_licenses                                     # [not win]
     # Clear out cache to avoid file not removable warnings
-    - chmod -R u+w $(go env GOPATH) && rm -r $(go env GOPATH)  # [unix]
+    - chmod -R u+w $(go env GOPATH) && rm -r $(go env GOPATH)                                        # [not win]
 
 requirements:
   build:
@@ -33,6 +38,18 @@ requirements:
 test:
   commands:
     - fzf --version
+    - if not exist %PREFIX%\\share\\fzf\\shell\\completion.bash exit 1    # [win]
+    - if not exist %PREFIX%\\share\\fzf\\shell\\completion.zsh exit 1     # [win]
+    - if not exist %PREFIX%\\share\\fzf\\shell\\key-bindings.bash exit 1  # [win]
+    - if not exist %PREFIX%\\share\\fzf\\shell\\key-bindings.fish exit 1  # [win]
+    - if not exist %PREFIX%\\share\\fzf\\shell\\key-bindings.zsh exit 1   # [win]
+    - if not exist %PREFIX%\\share\\fzf\\plugin\\fzf.vim exit 1           # [win]
+    - test -f ${PREFIX}/share/fzf/shell/completion.bash                   # [not win]
+    - test -f ${PREFIX}/share/fzf/shell/completion.zsh                    # [not win]
+    - test -f ${PREFIX}/share/fzf/shell/key-bindings.bash                 # [not win]
+    - test -f ${PREFIX}/share/fzf/shell/key-bindings.fish                 # [not win]
+    - test -f ${PREFIX}/share/fzf/shell/key-bindings.zsh                  # [not win]
+    - test -f ${PREFIX}/share/fzf/plugin/fzf.vim                          # [not win]
 
 about:
   home: https://{{ goname }}
@@ -42,6 +59,12 @@ about:
     - {{ pkg_src }}/LICENSE
     - library_licenses/
   summary: A command-line fuzzy finder
+  description: |
+    An interactive filter program for any kind of list; files, command history, processes, hostnames, bookmarks,
+    git commits, etc. It implements a "fuzzy" matching algorithm, so you can quickly type in patterns with omitted
+    characters and still get the results you want.
+  doc_url: https://{{ goname }}/blob/master/README.md
+  dev_url: https://{{ goname }}
 
 extra:
   recipe-maintainers:

